src/components/App.js

```diff
@@ -4,7 +4,7 @@ class App extends React.Component {
   render() {
     return (
       <div>
-        {process.env.NODE_ENV === 'development'? <h1>Hello dev World</h1> : <h1>Hello Production</h1>}
+        <h1>Hello World</h1>
```

Comment:
Also please add example for each loader (right now I only can see js). Add styles, images, etc.

Reply:
added each loader e.g css loader, image loader too.